src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/ExecutorFactory.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNetCore.Mvc.RazorPages.Internal
 {
     public static class ExecutorFactory
     {
-        public static Func<Page, object, Task<IActionResult>> CreateExecutor(
+        public static Func<Page, object, object[], Task<IActionResult>> CreateExecutor(
```

Comment:
Yup, `Func<object, object[], Task<IActionResult>>`. Basically drop the `Page` parameter since it's no longer useful for us to know about it.